Pack doesnt work for VB projects

## Details about Problem

NuGet product used VS UI 
VS version: Latest build from vsuwpt - 26323.2
Worked before? No

## Detailed repro steps so we can see the same problem

1. Create a VB .NET Standard class library (new template in vsuwpt).
2. Right-click the project and invoke Pack

Fails with this error:
```
1>ClassLibrary8 -> c:\users\srivatsn\documents\visual studio 2017\Projects\ClassLibrary8\ClassLibrary8\bin\Debug\netstandard1.4\ClassLibrary8.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\vsuwpt\Enterprise\MSBuild\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(104,5): error MSB4044: The "PackTask" task was not given a value for the required parameter "PackItem".
1>Done building project "ClassLibrary8.vbproj" -- FAILED.
```

--- a/src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets
+++ b/src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets
@@ -182,17 +182,12 @@ Copyright (c) .NET Foundation. All rights reserved.
   <!--
     ============================================================
     _LoadPackGraphEntryPoints
-    Find project entry points and load them into items.
+    Find project entry point and load them into items.
     ============================================================
   -->
   <Target Name="_LoadPackInputItems">
-    <!-- Allow overriding items with ProjectFileToPack -->
+    <!-- Allow overriding items with PackProjectInputFile -->
     <PropertyGroup Condition="'$(PackProjectInputFile)' == ''">
-      <PackProjectInputFile>$(ProjectFileToPack)</PackProjectInputFile>
-    </PropertyGroup>
-
-    <!-- Project case -->
-    <PropertyGroup Condition="$(MSBuildProjectFullPath.EndsWith('.csproj')) == 'true' AND '$(PackProjectInputFile)' == ''">
       <PackProjectInputFile>$(MSBuildProjectFullPath)</PackProjectInputFile>
     </PropertyGroup>
   </Target>

